src/utils/userInfo.ts

```diff
@@ -0,0 +1,17 @@
+import { USER_INFO } from '../constants/localStorage';
+import { IUserInfo } from '../types/auth';
+
+export const setUserInfo = (json: IUserInfo) => {
+  const userInfoString = JSON.stringify(json);
+  localStorage.setItem(USER_INFO, userInfoString);
+};
+
+export const getUserInfo = (): IUserInfo | undefined => {
+  const userInfo = localStorage.getItem(USER_INFO);
+  if (!userInfo) return undefined;
+  return JSON.parse(userInfo);
+};
```

Comment:
```suggestion
export const getUserInfo = (): IUserInfo | undefined => {
  const userInfoString = localStorage.getItem(USER_INFO);
  if (!userInfoString) return undefined;
  return JSON.parse(userInfoString);
};
```
뭔가 데드락 걸린 것 처럼, 위에는 userInfo로 얘는 userInfoString으로 바꿔달라하는데.. 
의미 상으로 userInfo의 JSON String이 맞으니까 이런 건 어떨까요?

Reply:
넹 수정했습니다₩!!~!